src/Launcher.vue

```diff
@@ -221,6 +228,26 @@ export default {
     disableUserListToggle: {
       type: Boolean,
       default: false
+    },
+    multipleChatsEnabled: {
+      type: Boolean,
+      default: false
+    },
+    chatList: {
+      type: Array,
+      required: false
+    },
+    chatListTitle: {
+      type: String,
+      default: () => ''
```

Comment:
why using a callable here instead of `default: ''`?

Reply:
I'm not sure, but I did it because that's how it was done in this file in many cases. I'm changing them all to just values.